docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -101,13 +154,101 @@ const theme = createTheme({
 });
 ```
 
+### Remove the keyboard view
+
+The picker components no longer have a keyboard view to render the input inside the modal on mobile.
+
+- If your date is easier to edit with the keyboard (e.g: a birthdate), you can directly use the new field components:
+
+  ```diff
+   function App() {
+     return (
+  -    <DatePicker />
+  +    <DateField />
+     )
+   }
+  ```
+
+- If you want users to be able to edit with both the keyboard using the input and with the touch using the views,
+  you can pass a custom `Layout` component slot to re-introduce the keyboard view.
+
+  :::warning
+  TODO: Add a demo in the "Custom layout" page to add it back.
+  :::
+
+  :::info
+  At some point, the mobile pickers should have a prop allowing to have an editable field without opening the modal.
+  :::
+
+### Change the DOM structure
```

Comment:
For these changes, maybe a few example with the DOM structure before / after would help
Because we don't have a 1 to 1 equivalent of DOM nodes since we reduced the nesting and moved to a grid approach.

We could create an example to add back the old DOM structure but I doubt it's a good solution.

Reply:
I also don't think that we should endorse keeping the previous/flawed DOM structure. 🤔 

But I'm leaning towards adding this "breaking change" into codemod umbrella issue and considering at least adding a tranformation doing what you've listed here. 🤔 
Would you agree that it would make sense?

Reply:
I don't know if it's worth it.
Because most of those changes are not purely 1 to 1 equivalent.
And I fear the codemod are going to be a massive pain building.